pilot/pkg/security/authz/model/principal_test.go

```diff
@@ -415,7 +415,51 @@ func TestPrincipal_Generate(t *testing.T) {
               - any: true`,
 		},
 		{
-			name: "principal with property attrSrcPrincipal for TCP filter",
+			name: "principal with property attrSrcPrincipal v1beta1",
+			principal: &Principal{
+				Properties: []KeyValues{
+					{
+						attrSrcPrincipal: []string{"id-1", "*", allAuthenticatedUsers, allUsers},
```

Comment:
I don't think we're using hard-coded strings "allAuthenticatedUsers" and "allUsers" anymore, due to some UX issue happened to the IAM team, so maybe don't use them in tests? 

cc @liminw

Reply:
It can be removed later

Reply:
We never officially supported allAuthenticatedUsers and allUsers. We can remove it in code.

Reply:
Should be removed in a separate PR.